banyan/src/forest/read.rs

```diff
@@ -167,7 +245,8 @@ where
                 Mode::Backward => self.offset.saturating_sub(head.index.count())..self.offset,
             };
```

Comment:
As I said previously: I’d rather have this twice, but in the right parts of the branches, than computing nonsense here.

Reply:
I agree.